addons/stock/report/stock_traceability.py

```diff
@@ -17,7 +18,7 @@ def autoIncrement():
     return rec
 
 
-class MrpStockReport(models.TransientModel):
+class StockTraceabilityReport(models.TransientModel):
```

Comment:
i woudln't change the class name in 13.0

Reply:
Fixed
(Not sure How does it break anything :thinking: )